cmd/snapshot.go

```diff
@@ -15,36 +15,42 @@
 package cmd
 
 import (
-	"fmt"
+	"time"
+	"strings"
+	"strconv"
 
+	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"golang.org/x/net/context"
 )
 
-// snapshotCmd represents the snapshot command
 var snapshotCmd = &cobra.Command{
 	Use:   "snapshot",
-	Short: "A brief description of your command",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Short: "Create a snapshot of a running KDK container",
+	Long:  `Create a snapshot of a running KDK container`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("snapshot called")
+		logger := logrus.New().WithField("command", "pull")
```

Comment:
Is it possible to run more than one KDK at a time?  Just one thing to keep in mind while you're coding this new.

Reply:
To clarify, it helps with developing the KDK, from within the KDK.

Reply:
Perhaps allowing the user to pass an alternative ID on the command line or ENV variable.  But, regularly default to an ID of "kdk".

Reply:
@dcwangmit01 good idea to enable running multiple KDK containers
This is not currently possible, but won't be difficult to implement.  

Please add an issue.